.travis.yml

```diff
@@ -13,6 +13,7 @@ env:
   global:
     - TRAVIS_JDK=11
     - JABBA_HOME=$HOME/.jabba    
+    - RUN_DROP_TRAVIS_CACHES_STAGE=true
```

Comment:
I will revert this to false, but first want to have at least one green build. 

Last time I created a custom build and set var to `true` it didn't work as expected. So forcing it now to move with this PR.

Reply:
Maybe instead of hardcoding the ENV_VAR we should remove this line andd set the ENV_VAR on the build settings in the Travis UI